### DISCUSSION TITLE
Multiple selectors

### DISCUSSION THREAD

**Discussion by edumaciel10** [eslint/eslint#16191] [General]
How can I find two CallExpressions selectors on my code using esquery?

Example of code
```js
// I want that every time functionA is called should also call functionB
functionA(); // get error if not call functionB
```

demo that I try: https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50IHF1b3RlczogW1wiZXJyb3JcIiwgXCJkb3VibGVcIl0gKi9cbi8vIFtcImVycm9yXCIse1wic2VsZWN0b3JcIjpcIkNhbGxFeHByZXNzaW9uW2NhbGxlZS5uYW1lPSdmdW5jdGlvbkEnXSAhQ2FsbEV4cHJlc3Npb25bY2FsbGVlLm5hbWU9J2Z1bmN0aW9uQSddXCIsXCJtZXNzYWdlXCI6XCJ5b3UgYWxzbyBzaG91bGQgY2FsbCBmdW5jdGlvbkJcIn1dXG5jb25zdCBmdW5jdGlvbkEgPSAoKSA9PiB7fTtcblxuZnVuY3Rpb25BKClcbiIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6ImxhdGVzdCIsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnsibm8tc2VsZi1hc3NpZ24iOlsiZXJyb3IiXSwibm8tc2V0dGVyLXJldHVybiI6WyJlcnJvciJdLCJuby1zaGFkb3ctcmVzdHJpY3RlZC1uYW1lcyI6WyJlcnJvciJdLCJuby1zcGFyc2UtYXJyYXlzIjpbImVycm9yIl0sIm5vLXRoaXMtYmVmb3JlLXN1cGVyIjpbImVycm9yIl0sIm5vLXVuZGVmIjpbImVycm9yIl0sIm5vLXVuZXhwZWN0ZWQtbXVsdGlsaW5lIjpbImVycm9yIl0sIm5vLXVucmVhY2hhYmxlIjpbImVycm9yIl0sIm5vLXVuc2FmZS1maW5hbGx5IjpbImVycm9yIl0sIm5vLXVuc2FmZS1uZWdhdGlvbiI6WyJlcnJvciJdLCJuby11bnNhZmUtb3B0aW9uYWwtY2hhaW5pbmciOlsiZXJyb3IiXSwibm8tdW51c2VkLWxhYmVscyI6WyJlcnJvciJdLCJuby11bnVzZWQtdmFycyI6WyJlcnJvciJdLCJuby11c2VsZXNzLWJhY2tyZWZlcmVuY2UiOlsiZXJyb3IiXSwibm8tdXNlbGVzcy1jYXRjaCI6WyJlcnJvciJdLCJuby11c2VsZXNzLWVzY2FwZSI6WyJlcnJvciJdLCJuby13aXRoIjpbImVycm9yIl0sInJlcXVpcmUteWllbGQiOlsiZXJyb3IiXSwidXNlLWlzbmFuIjpbImVycm9yIl0sInZhbGlkLXR5cGVvZiI6WyJlcnJvciJdLCJuby1yZXN0cmljdGVkLXN5bnRheCI6WyJlcnJvciIseyJzZWxlY3RvciI6IkNhbGxFeHByZXNzaW9uW2NhbGxlZS5uYW1lPSdmdW5jdGlvbkEnXSArIENhbGxFeHByZXNzaW9uW2NhbGxlZS5uYW1lPSdmdW5jdGlvbkInXSIsIm1lc3NhZ2UiOiJ5b3UgYWxzbyBzaG91bGQgY2FsbCBmdW5jdGlvbkIifV19LCJlbnYiOnsiZXM2Ijp0cnVlfX19

**Comment by nzakas** [comment]
I'm not entirely sure what you're asking. You can use `CallExpression` to capture all call expressions in your code. Then you'd need to look at the names of the expressions to determine what to do.

Try using astexplorer.net to look at the AST and see if that helps you.
